includes/login-mutation.php

```diff
@@ -65,7 +65,7 @@ function wpgraphql_cors_login_mutation() {
 					$user = wp_signon( $credentials, true );
 
 					if ( is_wp_error( $user ) ) {
-						throw new UserError( $user->get_message() );
+						throw new UserError( ! empty( $user->get_error_code() ) ? $user->get_error_code() : 'invalid login' ) );
```

Comment:
Should one of the two last brackets be removed?

Reply:
Great catch! Sorry about that 🤦 

Fixed.